src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2504,6 +2506,8 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "HealthCertificate_Details_certificate" = "Impfzertifikat";
 
+"HealthCertificate_Details_QRCode_Image_Description" = "QR-Code mit Ihrem Impfzertifikat %i von %i";
```

Comment:
```suggestion
"HealthCertificate_Details_QRCode_Image_Description" = "Ein QR-Code mit Ihrem Impfzertifikat %i von %i";
```

Reply:
@KaiTeuber to stay consistent with the other entries, can you please add the "Ein" as suggested. Thank you :)